AdminUi/src/AdminUi.Infrastructure/Persistence/SqlDatabaseConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace AdminUi.Infrastructure.Persistence;
+
+public class SqlDatabaseConfiguration
+{
+    public string Provider { get; set; } = string.Empty;
+    public string ConnectionString { get; set; } = string.Empty;
```

Comment:
Why didn't you add the validation annotations as we have it in all the database configuration of all the modules?